lib/modules/versioning/bazel-module/index.ts

```diff
@@ -0,0 +1,179 @@
+import type { SemVer } from 'semver';
+import type { NewValueConfig, VersioningApi } from '../types';
+import { BzlmodVersion } from './bzlmod-version';
+
+export const id = 'bazel-module';
+export const displayName = 'Bazel Module';
+export const urls = ['https://bazel.build/external/module'];
+export const supportsRanges = false;
+
+function getBzlmodVersion(version: string | SemVer): BzlmodVersion {
```

Comment:
```suggestion
function getBzlmodVersion(version: string): BzlmodVersion {
```
remove, we'll never pass a SemVer object